### DISCUSSION TITLE
Editor - universal (asterisk) selector renders preview as text

### DISCUSSION THREAD

**Discussion by weroro-sk** [cssbattle/forum#340] [Bugs reports]
Hello,
if I try to use universal (asterisk) selector with display property, the preview shows my css code as text.
In every battle task where I write `* {display:flex;}` is rendered as [see image below].
![Editor universal (asterisk) selector issue](https://i.ibb.co/WKpxyFS/Screenshot-2023-02-02-at-23-20-22.png)

Tested in Chrome, Safari, Firefox, Edge (mac os)

**Comment by aarthvaen** [comment]
You're applying `display:flex` on * selector which targets all your HTML elements, `<head>` included. If you use display property on *, the head will be shown in the preview.

You can use `* > *` or `*+*` to only select the body and use display property as you need.

**Reply by SelenIT** [reply]
`* > *` would still select `head` and its children like `style` 😳
